script/sendmail.sh

```diff
@@ -0,0 +1,65 @@
+#!/bin/bash
+
```

Comment:
Would you like to add author info? e.g. names/emails.

Reply:
I will add.

Reply:
You can modified this code to fit your needs as you see fit